<launch>
    <param name="/traj_planner/steps_per_phase" type="int" value="4" />
    <param name="/p_obstacle" type="double" value="0.1" />
    <param name="/n_obstacles" type="int" value="2" />
    <param name="/tree" type="bool" value="false" />
    <param name="/road_width" type="double" value="3.5" />
    <param name="/traj_controller/trajectory_index" type="int" value="1" />
    <param name="/traj_controller/low_pass_filter" type="boolean" value="true" />

    <node pkg="control_tree_car" type="obstacle_avoidance_tree_node" name="obstacle_avoidance" output="screen">
    </node>

    <node pkg="control_tree_car" type="trajectory_controller_node" name="trajectory_controller" output="screen">
    </node>

    <node pkg="control_tree_car" type="obstacle_popping_control_node" name="obstacle" output="screen">
    </node>

    <node pkg="control_tree_car" type="obstacle_avoidance_evaluation_node" name="evaluation" output="screen">
    </node>

    <include file="$(find control_tree_car)/launch/display_obstacle_avoidance.launch">
    </include>
</launch>
